tgui/packages/tgui/interfaces/PowerMonitor/Smes.tsx

```diff
@@ -55,31 +57,117 @@ export const PowerMonitorSmesGlobal = (_props, context) => {
   );
 };
 
-export const PowerMonitorSmesTableHeader = (props, context) => {
+interface PowerMonitorSemsTableHeaderProps {
+  setSortBy: (field: SmesTableHeaderColumns) => any;
+  state: SmesTableHeaderColumnSortState;
```

Comment:
Prefer calling `sortState` if passing it in as a prop; `state` could mean anything.

Reply:
agreed, state is a bad variable name, fixed!